cloud_run/shelf_server/README.md

```diff
@@ -0,0 +1,114 @@
+# Basic server example
+
+This example demonstrates a basic server using the [Shelf] package to handle
+HTTP GET requests. The server responds to requests at the root route (`/`)
+with:
+
+```
+Hello, World!
+```
+
+## Running the example
```

Comment:
As these instructions will be duplicated across all Cloud Run samples, it may be worth it to put an abridged version here and keep generic instructions in the `run/` folder. Eventually we'll move all README instructions to CloudSite.